src/shortcuts/template.spec.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { C } from '.';
+import { sql } from './template';
+
+describe('sql', () => {
+  it('works in basic case', () => {
+    expect(String(sql`SELECT * FROM tbl`)).toEqual(`SELECT * FROM tbl`);
+  });
+
+  it('works in complex case', () => {
+    const column = 'some_column';
+    const tbl = 'this is a "table" and I like it';
+    const v1 = 'foo';
+    const v2 = 'bar';
+    expect(
+      String(sql`SELECT * FROM "${tbl}" WHERE ${C(column)} = ${5} AND "x" IN (${v1}, '${v2}')`),
+    ).toEqual(
+      `SELECT * FROM "this is a ""table"" and I like it" WHERE "some_column" = 5 AND "x" IN ('foo', 'bar')`,
+    );
+  });
+
+  it('wrapping in single quotes forces a literal string', () => {
+    const c = C('blah');
```

Comment:
Are you deliberately using `C` here to demonstrate that it ignores the expression type, which in this case is SqlColumn?

Reply:
yes, I want to test that if wrapped in single quotes it will be forced to a string